lib/membrane/realtimer.ex

```diff
@@ -39,7 +44,24 @@ defmodule Membrane.Realtimer do
         tick_actions: [buffer: {:output, buffer}] ++ state.tick_actions
     }
 
-    {[timer_interval: {:timer, interval}], state}
+    {maybe_start_timer ++ [timer_interval: {:timer, interval}], state}
+  end
+
+  @impl true
+  def handle_event(:input, %Events.Reset{}, _ctx, state) do
+    {actions, state} =
+      case state.tick_actions do
+        [] when state.timer_status == :to_be_started ->
```

Comment:
How about warning here? It means that something went wrong, as the `Evenvts.Reset` event was delivered more than once in a row